backend/dal/Services/Admin/IAddressService.cs

```diff
@@ -0,0 +1,12 @@
+using System;
```

Comment:
Empty interface?

Reply:
For now yes.  Everything needs an interface if you want to mock it correctly.  But I don't have anything extra to add to this interface presently.